shared/fs/banner/conflict-banner-container.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+import * as React from 'react'
+import * as Constants from '../../constants/fs'
+import * as Types from '../../constants/types/fs'
+import {namedConnect} from '../../util/container'
+import ConflictBanner, {getHeight} from './conflict-banner'
+import * as RowTypes from '../row/types'
+import openUrl from '../../util/open-url'
+
+type OwnProps = {|path: Types.Path, conflictState: Types.ConflictState|}
+
+const mapStateToProps = (state, {path}: OwnProps) => ({
+  _tlf: Constants.getTlfFromPath(state.fs.tlfs, path),
+})
+
+const mapDispatchToProps = dispatch => ({
+  onFeedback: () => {},
+  onFinishResolving: () => {},
+  onHelp: () => openUrl('https://keybase.io/docs/kbfs/understanding_kbfs#conflict_resolution'),
+  onSeeOtherView: () => {},
```

Comment:
How about just do `onSeeServerView` and `onSeeLocalView` here? I think the explicit ones are easier to follow.